plugins/task-witness/controller/task_witness_deploy.py

```
@@ -6209,6 +6225,22 @@ def _validate_provider_source_and_modules(
+        and deployment_value["migration"].get("bridge_identity_sha256")
+        == BRIDGE_IDENTITY_SHA256
+    )
+    if (
+        receipt_contract == DEPLOYMENT_RECEIPT_CONTRACT
+        and "migration" not in deployment_value
+    ):
+        intrinsic_repository = "https://github.com/nisavid/provingkit"
+    elif bridge_migration or receipt_contract == LEGACY_DEPLOYMENT_RECEIPT_CONTRACT:
+        intrinsic_repository = "https://github.com/nisavid/agents"
+    else:
+        raise DeploymentError(f"{label} receipt contract is unsupported")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)